recipes/aimodelshare/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "aimodelshare" %}
+{% set version = "0.0.94" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 36b35968e0605fa57610eee39b568a491666ed4d76b5287ea8d276fb83294dd4
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - astunparse ==1.6.3
```

Comment:
I'm not against letting people shoot them selves in the foot, but equality pinnings should not be used for packages, rather they should be used in your environment files. can you consider releasing these as lower bound?

Reply:
I am sorry but I don't think it is possible for now, several time encountered issues on latest version therefore we deliberately specify the working version.

Reply:
Ok as long as you're aware of the limitations that imposes, I think we can merge this.